src/sage/modular/modform/eis_series_cython.pyx

```diff
@@ -140,7 +140,7 @@ cpdef Ek_ZZ(int k, int prec=10) noexcept:
     return val
 
 
-cpdef eisenstein_series_poly(int k, int prec = 10)  noexcept:
+cpdef eisenstein_series_poly(int k, int prec = 10) :
```

Comment:
```suggestion
cpdef eisenstein_series_poly(int k, int prec=10):
```

Reply:
I didn't make any change other than adding or removing `noexcept`.

We can add style changes all together in a single commit, or do you prefer to fixup the original commit?

Is there an easy way to run a linter/pycodestyle just in the lines that changed?

Reply:
only one so far <img width="154" alt="Screenshot 2024-03-25 at 9 31 59 AM" src="https://github.com/sagemath/sage/assets/8345221/680cb585-f589-412b-9ea0-5833063f56be">